serverless.yml

```diff
@@ -42,9 +56,5 @@ package:
     - "**.html"
 
 functions:
-  generateRecommendations:
-    handler: src.poprox_recommender.handler.generate_recs
-    events:
-      - httpApi:
-          path: /
-          method: post
+  # if offline mode enabled filePrefix = 'offline.' else filePrefix = ''
+  ${file(./serverless/${self:custom.offline.filePrefix}functions.yaml):functions}
```

Comment:
And then this selects between the contents of two different files depending on whether we're defining the functions for offline or deployment environments